src/Photos/Enums.cs

```diff
@@ -256,7 +281,14 @@ public enum PHAssetResourceType : long
 		FullSizeVideo = 6,
 		AdjustmentData = 7,
 		AdjustmentBasePhoto = 8,
-		PairedVideo = 9
+		[iOS (9,1)]
+		PairedVideo = 9,
+		[Mac (10,15, onlyOn64 : true), iOS (10,0)]
+		FullSizePairedVideo = 10,
+		[Mac (10,15, onlyOn64 : true), iOS (10,0)]
+		AdjustmentBasePairedVideo = 11,
```

Comment:
is it new in Xcode 11 ?
if so it's likely not available before iOS 13 (not 10)

Reply:
From the wiki it looks like it was already available in iOS 10, but the mac part is new. I'm not sure why it was not in the file previously

Reply:
indeed, seems it was forgotten last time
```
-    PHAssetResourceTypeAdjustmentBasePairedVideo PHOTOS_AVAILABLE_IOS_TVOS(10_0, 10_0) = 11,
```
^ here you see the _old_ macro was mentioning `tvOS` too, so you need a `[TV (10,0)]` on the members too

Reply:
Okay that makes sense!

Reply:
Just clarifying, since it is TV 10 and the class is decorated with TV 10, there's no need to put TV 10 on the methods themselves?